### DISCUSSION TITLE
Hotio mods

### DISCUSSION THREAD

**Discussion by lseufer** [themepark-dev/theme.park#155] [Q&A] [answered]
Me, again!

I am trying to apply the docker mods to the hotio readarr  image but have no idea how to do it. I have modified the environment variables like I did with the linuxserver ones but nothing happens.

Any guidance appreciated!

**Comment by GilbN** [comment, marked as answer]
docker mods only support lsio containers. But the scripts may support a hotio container, but you probably need to fix the paths in the scripts. The Scripts can be found in the docker-mods branch

https://github.com/gilbN/theme.park/tree/docker-mods#hotio-containers

**Reply by lseufer** [reply]
Thanks for the very swift reply. However, I have no idea how to call the script...

**Reply by GilbN** [reply]
You just download it, and add it to the hotio container

**Reply by GilbN** [reply]
https://hotio.dev/faq/#guides info here

**Reply by lseufer** [reply]
(Total noob here) so I had a volume to my container pointing to the script but do I need to add something to Environment? An ENTRYPOINT variable?

**Reply by GilbN** [reply]
No, should just need to mount it. That's all it says in the guide.

**Reply by lseufer** [reply]
So I tried mounting it as a folder and I get "unable to spawn /var/run/s6/etc/cont-init.d/99-my-script: Permission denied" any pointer?

**Reply by GilbN** [reply]
guide says to mount the file not folder

**Reply by lseufer** [reply]
I tried that before and it said something about trying to mount a file as a folder but now it works! Thanks a bunch
